src/content/en/tools/workbox/modules/workbox-strategies.md

```diff
@@ -136,7 +136,7 @@ workbox.routing.registerRoute(
   new RegExp('/images/'),
   workbox.strategies.cacheFirst({
     cacheName: 'image-cache'
-  }
+  })
```

Comment:
Can we put commas here (and below) while we're adding stuff :)

Reply:
As it turns out, looking through this again made me spot a bunch of references to `workbox.routing()` which were not ever the correct syntax. I've updated those in the latest commit.